deployer/scripts/heapster.sh

```diff
@@ -17,23 +17,6 @@
 #
 
 function deploy_heapster() {
```

Comment:
ansible changes: https://github.com/openshift/openshift-ansible/blob/master/roles/openshift_metrics/tasks/generate_heapster_certificates.yaml  but there is an outstanding PR that moves some of this logic to support a heapster only install